core/cog-view.c

```diff
@@ -103,10 +103,6 @@ static void
 cog_view_constructed (GObject *object)
 {
     G_OBJECT_CLASS (cog_view_parent_class)->constructed (object);
-
-    webkit_web_view_load_html (WEBKIT_WEB_VIEW (object),
-                               "<!DOCTYPE html><html><body></body></html>",
-                               "about:blank");
```

Comment:
Now that the `.constructed` callback does nothing, we can remove the whole `cog_view_constructed()` function.

Reply:
Updated. Please take in consideration these changes to be merged in Cog 0.10. Thanks.

Reply:
We don't need to do anything for 0.10 because `master` does not have the dummy load of `about:blank` :wink: